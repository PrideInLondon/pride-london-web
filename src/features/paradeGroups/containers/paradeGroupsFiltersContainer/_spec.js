import React from 'react'
import { shallow } from 'enzyme'
import ParadeGroupsFilterContainer from '.'

describe(ParadeGroupsFilterContainer.name, () => {
  it('should render correctly', () => {
    const handleFilterClick = jest.fn()

    const categories = [
      {
        title: 'All groups',
        hexColour: '#FFD95E',
      },
      {
        title: 'Arts',
        hexColour: '#FF5B44',
      },
    ]

    const wrapper = shallow(
      <ParadeGroupsFilterContainer
        selectedFilter={categories[0]}
        categories={categories}
        handleFilterClick={handleFilterClick}
      />
    )
    expect(wrapper).toMatchSnapshot()
  })
})
