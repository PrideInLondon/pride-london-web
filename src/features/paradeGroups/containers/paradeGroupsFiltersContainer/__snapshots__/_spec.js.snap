// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ParadeGroupsFilterContainer should render correctly 1`] = `
<styles__ParadeGroupsFiltersWrapper>
  <FilterButton
    filterName="paradeGroupsFilter"
    filterType={
      Object {
        "hexColour": "#FFD95E",
        "title": "All groups",
      }
    }
    handleClick={[MockFunction]}
    icon={null}
    isButton={true}
    isOutline={false}
    key="All groups"
  />
  <FilterButton
    filterName="paradeGroupsFilter"
    filterType={
      Object {
        "hexColour": "#FF5B44",
        "title": "Arts",
      }
    }
    handleClick={[MockFunction]}
    icon={null}
    isButton={true}
    isOutline={true}
    key="Arts"
  />
</styles__ParadeGroupsFiltersWrapper>
`;
